### ISSUE TITLE
Document use of custom templates

### ISSUE THREAD

**Issue by rennerocha** [scrapinghub/spidermon#135]
Related to #126 , we need a better documentation to explain how to define and use custom templates for e-mails, reports, etc.

[This action](https://github.com/scrapinghub/spidermon/blob/master/spidermon/contrib/actions/templates.py) contains part of the code that handles it. 

Include examples in the tutorial would be nice as well.

**Comment by vipulgupta2048** [comment]
Yes, thank you for following up on that. If no one takes this up, I would be more than happy to contribute an example to the docs. Since I am working on Slack action as well, it would be great to learn more about other actions.
